tcf_website/static/landing/feedbackform.js

```diff
@@ -1,34 +1,18 @@
 import { validateForm } from "../common/form.js";
+import { postToDiscord, sendEmail } from "../common/send_feedback.js";
 
 function submit(event) {
-    var form = document.getElementById("feedbackform");
-    var valid = validateForm(form);
-    if (valid === true) {
-        postToDiscord(event);
-    }
-}
-
-function postToDiscord(event) {
-    var fname = $("#inputFname").val();
-    var lname = $("#inputLname").val();
-    var email = $("#inputEmail").val();
-    var title = $("#inputTitle").val();
-    var message = $("#inputMessage").val();
+    const form = document.getElementById("feedbackform");
+    if (validateForm(form) === true) {
+        const data = new FormData(form);
+        const content = Object.fromEntries(data.entries());
+        const type = "feedback";
 
-    var data = {
-        type: "feedback",
-        content: "Feedback submitted" +
-        "\n**Name:** " + fname + " " + lname +
-        "\n**Email:** " + email +
-        "\n**Title:** " + title +
-        "\n**Message:** \n" + message
-    };
-
-    $.ajax({
-        type: "GET",
-        url: "/discord/",
-        data: data
-    });
+        postToDiscord(type, content);
+        if (content.email !== "") {
```

Comment:
I don't know if there are other ways to handle email validation because it seems like some emails will pass, but will cause errors when trying to actually send the email (image below) I'm not sure if this is an elegant solution but could the form have an opt-in system for emails and then in the backend we take the email from the account logged in? I guess this would have some issues with the feedback page since that doesn't require login though.
![image](https://user-images.githubusercontent.com/26658168/155891908-66590d39-427d-4468-84b2-35eff081bc22.png)